client/src/pages/ProfileListings/ProfilePreview/ProfilePreview.tsx

```diff
@@ -0,0 +1,54 @@
+import { Card, CardContent, CardActions, IconButton, Typography, Avatar } from '@material-ui/core';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import Rating from '@material-ui/lab/Rating';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import useStyles from './useStyles';
+
+interface PreviewProps {
+  img: string;
+  name: string;
+  subtitle: string;
+  rating: number;
+  description: string;
+  location: string;
+  payRate: number;
+}
+
+const ProfilePreview = ({ img, name, subtitle, rating, description, location, payRate }: PreviewProps): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.root}>
+      <CardContent className={classes.content}>
```

Comment:
```suggestion
      <CardActionArea comopnent={Link} to=`/profile-details/${profile.id}`
      <CardContent className={classes.content}>
```
Adding the cardaction area here to navigate from the listing page to the details page would a nice tough 👍🏽